### PR TITLE
Rtf - IFI assignment

### PR DESCRIPTION
**Pull request by S-Dafarra**
@francesco-romano @traversaro @pattacini

### CODE REVIEW

**Comment by traversaro** [comment]
You can explicitly request a review in GitHub : 
https://help.github.com/articles/requesting-a-pull-request-review/

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8)
+
+add_subdirectory(src/)
```
Comment (traversaro) [inline CMakeLists.txt]: In general I always see add_subdirectory arguments without the path separator / . I think for consistency is good to omit it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+option(COMPILE_TEST "Compile Tests" ON)
+if(COMPILE_TEST)
```
Comment (traversaro) [inline CMakeLists.txt]: There is a general purpose CMake option called BUILD_TESTING ( https://cmake.org/cmake/help/v3.1/module/CTest.html ).

**Review by traversaro — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (traversaro) [reply CMakeLists.txt]: It is convenient to use that so that all CMake projects use the same option.

**Comment by pattacini** [comment]
@S-Dafarra this is a super old PR that still appears on my personal dashboard.
Shall we close it?

**Comment by S-Dafarra** [comment]
Still working on it 😂

**Pull request closed without merging**